src/fbw/src/FlyByWireInterface.cpp

```diff
@@ -764,6 +773,12 @@ bool FlyByWireInterface::updateAdditionalData(double sampleTime) {
   additionalData.tillerHandlePosition = idTillerHandlePosition->get();
   additionalData.noseWheelPosition = idNoseWheelPosition->get();
 
+  additionalData.syncFoEfisEnabled = idSyncFoEfisEnabled->get() == 1;
```

Comment:
This seems to be unnecessary type conversions. You want a double (see other comment in .h) so no need for == 1 (bool) - also the next 3 lines